tests/bazelignore/REPO.bazel

```diff
@@ -0,0 +1,13 @@
+repo()
+
+# A single ignore_directories() with various comments and syntaxes
+ignore_directories([
+    "repo1",
+    # single line comment within array
+    "**/repo2",
+    "**/repo3/**", # trailing comment within array and doublestars
```

Comment:
I didn't notice until now that we have no test for what I believe to be the most common usage: a pattern with a non-recursive `*` wildcard such as `sub/bazel-*`.

Reply:
Would `sub*/repo4-*` cover that? See latest push.

I thought things like `**/out` or `**/node_modules` would be the most common ones, maybe also basics like just `ci-scripts` etc.

Reply:
It does. You are right, those are likely even more common, but `bazel-*` was the original motivation for this feature.